library/cwm/src/lib/cwm/abstract_widget.rb

```diff
@@ -83,6 +83,15 @@ def self.widget_type=(type)
     # @!method help
     #   @return [String] translated help text for the widget
 
+    # @!method dynamic_help
+    #   Dynamic help is useful when content of help text is not static and when
+    #   help is recomputed with {Yast::CWM#ReplaceWidgetHelp}, then it can
+    #   return different result. Examples where it is useful is e.g. replace point which
+    #   have dynamic content and different help text depending on what is displayed.
+    #   Another example is CWM::Table which can have dynamic header depending on conditions.
+    #   In those cases after call of {Yast::CWM#ReplaceWidgetHelp}, it will return new help text
+    #   @return [String] translated help text for the widget.
```

Comment:
@mvidner tricky part is that it is possible that dynamic help can change as side-effect of another action. Like consider that you have partition. In some conditions, it can have RadioButtons with 4 options, but in other when e.g. filesystem changed to btrfs, then it will have 6 options, so help text will change. In this case only part you need to do is to call `ReplaceWidgetHelp` in filesystem handle and not in RadioButtons itself.

And that it is exactly what happen with replace point. As outer replace point help change as a resulf of change of inner one, but outer one do not know about it. So only inner one trigger "help can change" and it is recomputed.